backend/app/views/spree/admin/payments/new.html.erb

```diff
@@ -21,6 +21,6 @@
     </fieldset>
   <% end %>
 <% else %>
-  <%= Spree.t(:cannot_create_payment_without_payment_methods) %>
-  <%= link_to Spree.t(:please_define_payment_methods), admin_payment_methods_url %>
+  <%= Spree.t(:cannot_create_payment_without_payment_methods_html,
+              link:(link_to Spree.t(:cannot_create_payment_link), admin_payment_methods_url)) %>
```

Comment:
I felt that by combining it we don't take any assumptions about the grammar of a language someone might be using.  Simply shows that we want to provide a link to creating a new payment method and a note explaining that there currently are none.